### PR TITLE
Show all secrets on the Secrets page

### PR DESCRIPTION
**Pull request by jwforres**
Now that we do not include Secret in the Other Resources page there has to be a way to get to all of them

### CODE REVIEW

**Comment by jwforres** [comment]
all-secrets

**Comment by jwforres** [comment]
@spadgett PTAL

**Comment by jwforres** [comment]
just noticed a typo on the create secret page, including that fix with this PR

File: app/views/secrets.html [hunk 1]
```diff
@@ -73,6 +73,31 @@
...
+                <h3>Other Secrets</h3>
+                <table class="table table-bordered table-hover table-mobile secrets-table">
+                  <thead>
+                    <tr>
+                      <th>Name</th>
+                      <th>Type</th>
+                      <th>Created</th>
+                    </tr>
+                  </thead>
+                  <tbody ng-repeat="secret in secretsByType.other | orderBy : 'name'">
```
Comment (spadgett) [inline app/views/secrets.html]: Should this be `'metadata.name'`? The order in the screenshot looks right though.

**Comment by spadgett** [comment]
One question, otherwise LGTM

**Comment by jwforres** [comment]
fixed [merge]

**Pull request merged**